pkgs/darwin-uninstaller/default.nix

```diff
@@ -62,10 +51,7 @@ stdenv.mkDerivation {
         esac
     fi
 
-    export nix=${nix}
-    export NIX_PATH=${nixPath}
-    system=$($nix/bin/nix-build '<darwin>' -A system)
-    $system/sw/bin/darwin-rebuild switch
+    ${uninstallSystem.system}/sw/bin/darwin-rebuild activate
```

Comment:
Is it sufficient to activate this profile, instead of using `switch`? `activate` really wants to build.

Reply:
Yeah, last time I checked `activate` is `switch` without the `build`